### PR TITLE
My order

### PR DESCRIPTION
**Pull request by MarDatsko**
overridden method toString for Order, Product, PositionItem added opportunity print user order in window MyOrder

### CODE REVIEW

File: InternetShop/src/main/java/model/Order.java [hunk 1]
```diff
@@ -31,22 +39,39 @@ public long getId() {
         return itemList;
     }
 
+    public void printAllItem(){
+        List<PositionItem> m = new ArrayList<>(getItemList());
```
Comment (lazermann111) [inline InternetShop/src/main/java/model/Order.java]: no need to create array list here

File: InternetShop/src/main/java/view/impl/StartMenu.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package view.impl;
+
+import view.Menu;
+
+import java.util.Scanner;
+
+public class StartMenu implements Menu {
+    private RegistrationMenu regMenu = new RegistrationMenu();
```
Comment (lazermann111) [inline InternetShop/src/main/java/view/impl/StartMenu.java]: delete these menu, we have one in staticMenus

File: InternetShop/src/main/java/view/impl/OrderCheckoutMenu.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Override
+    public void show() {
+        System.out.println("Order checkout");
+        checkOrder(scanner);
+    }
+
+    @Override
+    public void back() {
+
```
Comment (lazermann111) [inline InternetShop/src/main/java/view/impl/OrderCheckoutMenu.java]: plz add implementation for all exit() and back() in all menus

**Review by lazermann111 — CHANGES_REQUESTED** [review]
plz fix comments

**Review by lazermann111 — APPROVED** [review]

**Review by Ronashovkova — APPROVED** [review]

**Pull request merged**